subprojects/docs/src/docs/userguide/installation.adoc

```diff
@@ -15,45 +15,148 @@
+----
 
-Gradle requires a Java JDK or JRE to be installed, version 7 or higher (to check, use `java -version`). Gradle ships with its own Groovy library, therefore Groovy does not need to be installed. Any existing Groovy installation is ignored by Gradle.
+Gradle ships with its own Groovy library, therefore Groovy does not need to be installed. Any existing Groovy installation is ignored by Gradle.
 
 Gradle uses whatever JDK it finds in your path. Alternatively, you can set the `JAVA_HOME` environment variable to point to the installation directory of the desired JDK.
 
-[[sec:download]]
-=== Download
+=== Installing with a package manager
+
+link:http://sdkman.io[SDKMAN!] is a tool for managing parallel versions of multiple Software Development Kits on most Unix-based systems.
+
+----
+❯ sdk install gradle 4.6
+----
+
+link:http://brew.sh[Homebrew] is "the missing package manager for macOS".
+
+----
+❯ brew install gradle
+----
+
+link:http://scoop.sh[Scoop] is a command-line installer for Windows inspired by Homebrew.
+
+----
+❯ scoop install gradle
+----
+
+link:https://chocolatey.org[Chocolatey] is "the package manager for Windows".
+
+----
+❯ choco install gradle
+----
+
+link:https://www.macports.org[MacPorts] is a system for managing tools on macOS:
+
+----
+❯ sudo port install gradle
+----
+
+<<sec:installation_next_steps,↓ Proceed to next steps>>
+
+
+=== Installing manually
+
+==== Step 1. link:https://gradle.org/releases[Download] the latest Gradle distribution
+
+The distribution ZIP file comes in two flavors:
+
+ - Binary-only (bin)
+ - Complete (all) with docs and sources
 
-You can download one of the Gradle distributions from the {website}/downloads[Gradle web site].
+Need to work with an older version? See the link:https://gradle.org/releases[releases page].
 
-[[sec:unpacking]]
-=== Unpacking
+==== Step 2. Unpack the distribution
 
-The Gradle distribution comes packaged as a ZIP. The full distribution contains:
+===== Linux & MacOS users
 
-* The Gradle binaries.
-* The user guide (HTML and PDF).
-* The DSL reference guide.
-* The API documentation (Javadoc).
-* Extensive samples, including the examples referenced in the user guide, along with some complete and more complex builds you can use as a starting point for your own build.
-* The binary sources. This is for reference only. If you want to build Gradle you need to download the source distribution or checkout the sources from the source repository. See the {website}/development[Gradle web site] for details.
+Unzip the distribution zip file in the directory of your choosing, e.g.:
 
+----
+❯ mkdir /opt/gradle
+❯ unzip -d /opt/gradle gradle-4.6-bin.zip
+❯ ls /opt/gradle/gradle-4.6
+LICENSE  NOTICE  bin  getting-started.html  init.d  lib  media
+----
 
-[[sec:installation_environment_variables]]
-=== Environment variables
+===== Microsoft Windows users
+
+Create a new directory `C:\Gradle` with **File Explorer**.
```

Comment:
Should we rather propose to create the Gradle installation under the user home dir? I am not quite sure what the best practice looks like for Windows.

Reply:
Neither am I, but this is copy I just reused from the existing installation doc.